### PR TITLE
ntex framework

### PR DESCRIPTION
**Pull request by fafhrd91**
Add ntex framework benchmark

### CODE REVIEW

**Comment by joanhey** [comment]
I think that we need to enforce all the url names.
Check 
https://github.com/TechEmpower/FrameworkBenchmarks/pull/5710/files#diff-1cd02081fa7857523980ad89891563d4

@nbrady-techempower  What do you think ?

**Comment by nbrady-techempower** [comment]
@joanhey Agreed.

This change will be coming in #5648 and will break frameworks not doing it.

I'm going to get that merged in this week so @fafhrd91 could you please update this to use `/json`, `/plaintext`, etc.

This change for Round 20 is also noted here: #5709

**Comment by fafhrd91** [comment]
updated

File: frameworks/Rust/ntex/benchmark_config.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      "database_os": "Linux",
+      "display_name": "ntex",
+      "notes": "",
+      "versus": ""
+    },
+    "db": {
+      "fortune_url": "/f",
+      "db_url": "/d",
+      "query_url": "/q?q=",
+      "update_url": "/u?q=",
```
Comment (joanhey) [inline frameworks/Rust/ntex/benchmark_config.json:26]: Also /fortune, /db, /query and /update.

Comment (fafhrd91) [reply frameworks/Rust/ntex/benchmark_config.json:26]: this should be reflected in ticket

Comment (fafhrd91) [reply frameworks/Rust/ntex/benchmark_config.json:26]: @nbrady-techempower are those mandatory paths as well?

Comment (nbrady-techempower) [reply frameworks/Rust/ntex/benchmark_config.json:26]: @fafhrd91 Sorry, yes. I should have made that clearer. The verification changes in #5648 account for minimum required lengths for all routes.

Comment (fafhrd91) [reply frameworks/Rust/ntex/benchmark_config.json]: sure, np. Updated

**Comment by botika** [comment]
@fafhrd91 uses `yarte@0.9.3` which integrates the rendering of numbers by `itoa` and `dtoa` with a considerable performance improvement.

**Comment by fafhrd91** [comment]
I will, but I need base bench numbers first

**Comment by botika** [comment]
Sorry, [benchs](https://github.com/botika/template-bench-rs) and [results](https://travis-ci.org/github/botika/template-bench-rs). The difference in Big table bench is around `60us`

**Comment by fafhrd91** [comment]
I need base numbers for this bench

**Comment by fafhrd91** [comment]
@botika I will switch to yarte in next pr. is there way to render yarte template to `io::Write`?

**Comment by botika** [comment]
@fafhrd91  I suppose you mean something similar to [this](https://github.com/botika/yarte/blob/master/benches/src/all.rs#L216-L271), I can do it without problem and set the maximum size for the template at compile time, if you mean something else, open an issue or a PR and I will gladly integrate it, even if it is not entirely safe.

**Pull request merged**